docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -104,7 +104,7 @@ Running binaries other than `sui` might require installing prerequisites itemize
 
 To confirm that Sui installed correctly, type `sui --version` in your console or terminal and press Enter. The response should provide the Sui version installed. If the console or terminal responds with a command not found error, make sure the full path to your Sui binaries is included in your `PATH` variable.
 
-### Install Sui binaries from source {#install-sui-binaries-from-source}
+## Install from Cargo {#install-sui-binaries-from-source}
```

Comment:
Sorry for the confusion on where this section should go @ronny-mysten -- in the previous PR, I saw "Sui binaries" in both titles and assumed they were related, but they aren't really, so this PR is an effort to clarify that.